docs/decisions/0004-downstream-effect-events.rst

```diff
@@ -0,0 +1,103 @@
+Implementation of Event Driven Architecture for Exam Downstream Effects
+=======================================================================
+
+Status
+------
+
+Draft (circa Sept 2023)
+
+Context
+-------
+For Milestone 6 of the exams IDA project, we are porting any downstream effects of exam submission and review
```

Comment:
Nit. Since the audience for this ADR is 2U and Open edX, I don't think we need to make reference to internal concepts like milestones here. It may be confusing, and it's not relevant to the actual subject of the ADR.